BEPs/BEP-319.md

```diff
@@ -0,0 +1,91 @@
+      - [4.1.1 Existing Allocation Logic](#411-existing-allocation-logic)
+      - [4.1.2 Allocation Logic Moved to the Contract](#412-allocation-logic-moved-to-the-contract)
+      - [4.1.3 Maintain Burn Ratio](#413-maintain-burn-ratio)
+  - [4.2 Balancing Rewards](#42-balancing-rewards)
+  - [4.3 Extending Submission Deadline for Malicious Voting Evidence](#43-extending-submission-deadline-for-malicious-voting-evidence)
+  - [5. License](#5-license)
+
+<!-- /code_chunk_output -->
+
+
+
+## 1. Summary 
+This BEP proposes three optimizations to the incentive mechanism of the Fast Finality feature in BSC.
+
+## 2. Abstract 
+The proposal focuses on three key optimizations for the incentive mechanism of the Fast Finality feature:
+1. Making Fast Finality rewards governable while maintaining the basic block transaction fee allocation ratio.
+2. Ensuring a more balanced distribution of Fast Finality rewards between epochs.
+3. Extending the submission deadline for malicious voting evidence.
+
+## 3. Motivation 
+The introduction of the Fast Finality feature in BSC significantly reduces the time required for transaction final confirmation. This feature represents a substantial advantage for BSC compared to other EVM-compatible chains. The purpose of this BEP is to solidify and enhance this advantage.
+
+## 4. Specification
+### 4.1 Governable Rewards
+Currently, the reward is approximately 1/16 of the block transaction fees, leading to a lack of incentive for some validators to participate in voting, thereby reducing the stability expectation of the Fast Finality feature. This BEP makes Fast Finality rewards governable, laying the groundwork for future reward enhancements.
+#### 4.1.1 Existing Allocation Logic
+<div align="center">
+<img src=./assets/BEP-319/4.1-current-fee-distribution.png width=70% />
+</div>
+The existing allocation steps are as follows:
+
+1. Validators collect transaction fees from within the block.
+2. In the client, if the balance of the SystemRewardContract is less than 100 BNB, 1/16 of the collected transaction fees is allocated as rewards for relayers and Fast Finality voting. The remainder is deposited into the ValidatorContract.
+3. Within the ValidatorContract, 10% of the deposited amount is burned directly, and the remainder serves as potential earnings for validators and stakers.
+
+#### 4.1.2 Allocation Logic Moved to the Contract 
+<div align="center">
+<img src=./assets/BEP-319/4.2-new-fee-distribution.png width=90% />
+</div>
+Due to the deposition of the SystemRewardContract being in the client, requiring hard forks for each change, and for ease of governance, it is moved to the smart contract. The adjusted allocation logic is as follows:
+
+1. Validators collect transaction fees from within the block and deposit all into the ValidatorContract.
+2. Deposit finalityRewardRatio of transaction fees into the SystemRewardContract, where finalityRewardRatio is governable, with an initial value of 1/16.
```

Comment:
I would suggestion change the name: `finalityRewardRatio` to `systemRewardRatio`, as it is for relayer and finality

Reply:
accepted

Reply:
accept